src/screens/Forms/index.tsx

```diff
@@ -127,9 +127,11 @@ const Forms = () => {
         return (res.data?.me?.projects || []) as ProjectType[];
     }, [res.data]);
 
-    useEffect(() => {
-        refetch();
-    }, [refetch]);
+    useFocusEffect(
+        useCallback(() => {
```

Comment:
is usecallback hook needed inside other hook?

Reply:
implemented as suggested in react natvition [doc](https://reactnavigation.org/docs/use-focus-effect/)